Tests/configure_and_test_integration_instances.py

```diff
@@ -29,7 +29,7 @@ def options_handler():
     parser.add_argument('-p', '--password', help='The password for the login', required=True)
     parser.add_argument('--ami_env', help='The AMI environment for the current run. Options are '
                                           '"Server Master", "Demisto GA", "Demisto one before GA", "Demisto two before '
-                                          'GA". The server url is determined by the AMI environment.')
+                                          'GA", "Demisto PreGA". The server url is determined by the AMI environment.')
```

Comment:
i'll revert this one as it's only description and we will remove it once 5.5 is GA